src/rpdk/core/test.py

```diff
@@ -320,6 +320,7 @@ def get_contract_plugin_client(args, project, overrides, inputs):
             project.type_name,
             args.log_group_name,
             args.log_role_arn,
+            headers=(args.source_account, args.source_arn),
```

Comment:
I believe headers will always not be None and have a length of 2 in this case (None, None) so it would always go into the inject_headers in boto_helpers.py

Reply:
why do you think this?

Reply:
got it now :)